lib/workers/repository/updates/flatten.ts

```diff
@@ -216,8 +217,21 @@ export async function flattenUpdates(
       update.semanticCommits = semanticCommits;
     }
   }
-  return updates
-    .filter((update) => update.enabled)
+  let filteredOutCount = 0;
+  const filteredUpdates = updates
+    .filter((update) => {
+      if (update.enabled === false) {
+        filteredOutCount += 1;
+        return false;
+      }
+      return true;
+    })
     .map(({ vulnerabilityAlerts, ...update }) => update)
     .map((update) => filterConfig(update, 'branch'));
+  if (filteredOutCount) {
```

Comment:
why not simply
```suggestion
  if (filteredUpadates.length <  updates.length) {
```
so above becomes simpler.